packages/story-editor/src/components/footer/layers/layers.js

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState } from '@googleforcreators/react';
+import { __ } from '@googleforcreators/i18n';
+import {
+  Icons,
+  Text,
+  NotificationBubble,
+  THEME_CONSTANTS,
+  BUBBLE_VARIANTS,
+} from '@googleforcreators/design-system';
+
+/**
+ * Internal dependencies
+ */
+import styled, { css } from 'styled-components';
+import { LayerPanel } from '../../panels/design';
+import useLayers from '../../panels/design/layer/useLayers';
+import Popup from '../../secondaryPopup';
+
+const Container = styled.div`
+  position: absolute;
+  right: 0;
+  bottom: 0;
+  width: 300px;
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
+  overflow: auto;
+`;
+
+export const LabelText = styled(Text)`
+  color: ${({ theme }) => theme.colors.fg.primary};
+  padding-right: 8px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: hidden;
+`;
+
+const Button = styled.button`
+  border: 1px solid ${({ theme }) => theme.colors.border.defaultNormal};
+  background-color: ${({ theme }) => theme.colors.opacity.footprint};
+  border-radius: ${({ theme }) => theme.borders.radius.small};
+  padding: 5px 8px;
+  cursor: pointer;
+  display: flex;
+`;
+
+export const ChevronWrap = styled.div(
+  ({ theme, isOpen }) => css`
+    color: ${theme.colors.fg.primary};
+    width: 32px;
+    height: 32px;
+    margin: -4px;
+
+    ${isOpen &&
+    css`
+      transform: rotate(180deg);
+    `}
+  `
+);
+
+function Layers() {
+  const layers = useLayers();
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <>
+      <Popup isOpen={isOpen} anchor="right" zIndex={9999}>
```

Comment:
setting the zIndex here to 9999 is alot. I'm addressing the zIndex here: https://github.com/GoogleForCreators/web-stories-wp/pull/10861 - following this format would be very helpful. i'm currently blocked by Karma tests from merging it. You can follow the same pattern as the other footer popups. I'll write up more specifically what this is here shortly.

Reply:
`Number.MAX_SAFE_INTEGER` **FTW** 🚀 🌔 .

Reply:
I thought `9999` is some kind of magic number, we use it a lot 😉  (I guess it just spread like a virus)

<img width="826" alt="image" src="https://user-images.githubusercontent.com/21036224/157946850-5f0de92e-ce07-4f24-8d3f-8cb97b7a2bf3.png">